config/initializers/node-orm2-connect.js

```diff
@@ -15,9 +15,5 @@ export default {
       application.logger.error('Error initializing the node-orm2 adapter or database connection:');
       application.logger.error(error.stack);
     }
-
-    if (config.syncSchema) {
-      await fromNode((cb) => adapter.db.sync(cb));
-    }
```

Comment:
the way orm2 works, you have to follow this flow:

Connect to DB -> define models -> sync database schema.

The way the initializer was defined previously it was doing:

Connect to DB -> sync database schema -> define models